src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -56,19 +56,20 @@ public abstract class PlatformInstallerBase
                 .AppendLine("PLATFORM_BINARY_DOWNLOAD_START=$SECONDS")
                 .AppendLine($"platformName=\"{platformName}\"")
 
+                // TODO: PR2 remove this if statement, as bullseye images will be supported
                 // Temporarily use buster SDKs in bullseye image until bullseye SDKs are supported in storage account.
                 // Bug item: 1578477
-                .AppendLine($"if [ \"$DEBIAN_FLAVOR\" = \"bullseye\" ]; then")
-                .AppendLine($"SDK_DEBIAN_TYPE=\"buster\"")
+                .AppendLine($"if [ \"$DEBIAN_FLAVOR\" = \"{OsTypes.DebianBullseye}\" ]; then")
+                .AppendLine($"SDK_DEBIAN_TYPE=\"{OsTypes.DebianBuster}\"")
```

Comment:
`SDK_DEBIAN_TYPE` can be removed and use only `DEBIAN_FLAVOR` in the file.

Reply:
I'll remove this in PR2, when we completely remove the logic that keeps Bullseye from pulling its own SDKs